### DISCUSSION TITLE
Large export issue

### DISCUSSION THREAD

**Discussion by carava1998** [AntlerVC/firetable#1281] [Support / Q&A]
When I click on the enable button and after enabling Rowy Run, i get this message:

Failed to create Cloud Run Job: Violation in CreateJobRequest.job.annotations[run.googleapis.com/launch-stage]: system annotations are not supported in Cloud Run API v2.

**Comment by geckl** [comment]
@carava1998 did you ever find a solution to this issue? Stuck on the same thing myself

**Comment by BeeBombshell** [comment]
Hey @carava1998, @geckl... Could you share a screen recording via https://www.jam.dev so that we can figure out the root cause of this issue?

**Reply by geckl** [reply]
Hi @BeeBombshell here's a jam link, thanks for your help!

https://jam.dev/c/fef49639-dd50-4a58-aa56-0e7487ecda1a

**Reply by BeeBombshell** [reply]
Thank you for the recording, @geckl! Our team is looking into it and hopefully will be fixed soon. We'll keep you posted...

**Comment by BeeBombshell** [comment]
Hey @carava1998, @geckl. sorry about the bug in our Full-Text Search feature. We're on it and trying to fix it as soon as possible. We'd like to offer you some PRO credits as compensation to make it up to you. Can you share your project ID and email so we can sort this out for you? 😊

**Reply by carava1998** [reply]
Hi, my email is cesar@leadsales.io, but I can't find my project ID

**Comment by bofeiw** [comment]
This issue is fixed on rowy.app. Please let me know if the issue persists. @carava1998, @geckl
